py/examples/tour.py

```diff
@@ -220,31 +222,49 @@ def get_wave_completions(line, character, file_content):
         scripts=[ui.script(q.app.tour_assets + '/loader.min.js')],
         script=ui.inline_script(content=template, requires=['require'], targets=['monaco-editor']),
         layouts=[
-            ui.layout(breakpoint='xs', zones=[
+            ui.layout(
+                breakpoint='xs',
+                zones=[
+                    ui.zone('mobile_header'),
+                    ui.zone('mobile_blurb'),
+                    ui.zone('main', size=f'calc(100vh - {header_height + mobile_blurb_height}px)', direction=ui.ZoneDirection.COLUMN, zones=[
+                        ui.zone('code', size='50vh'),
+                        ui.zone('preview', size='50vh'),
+                    ])
```

Comment:
It's only for avoiding "magic" numbers.

Reply:
> It's only for avoiding "magic" numbers.

The question is why the calculation (regardless of with or without magic numbers) is necessary here. It's suspicious since the code calculates the size of the parent (main) zone and then ignores it completely by setting both children to `50vh`, but maybe I am missing something?

Reply:
How is it fixed if both calculations (parent and children) are still there?

Reply:
You are right, the parent (`main`) calculation is redundant. Removed.